docs/getting-started.md

```diff
@@ -193,77 +207,82 @@ data will be lost. Keep in mind, that it may take a much longer time from 8V to
 
 {% include end-panel.html %}
 
-{% include end-section.html %}
+<div class="row">
+<div class="col-md-8" markdown="1">
+### Shutting down
+
+You'll often want to turn off your EV3 while it isn't in use. When you are ready
+to wish your EV3 good night, turn if off by pressing the *back* button from the
+main menu in brickman or pressing _and holding_ the *back* button from any screen
+in brickman. This will open a dialog where you can select *Power Off* to turn
+off the EV3. _If you're using the command line, running `sudo poweroff` in a
+terminal works too._
+</div>
+
+<div class="col-md-4">
+{% include screenshot.html source="/images/brickman/shutdown-menu.png" caption="The 'Shutdown...' dialog in Brickman" %}
+</div>
+</div>
 
+{% include end-section.html %}
 
 {% include begin-section.html bg="dark" %}
 
 ## Step 5: Set up a network connection
 
-There are lots of choices here. Your choice depends on what type of connection
-you want to use and on the OS of your host computer, so pick the one that applies:
-
-* __Ethernet over USB__ (just requires the USB cable that comes with the EV3)
-    * [Connecting to the Internet](../tutorials/connecting-to-the-internet-via-usb) tutorial
-* __USB Ethernet dongle__ (as in the kind with an RJ45 connector)
-    * If your network has a DHCP server, this will "just work".
-* __USB Wi-Fi Dongle__
-    * Setting up Wi-Fi Via the Brickman User Interface (hopefully you can figure
-      this out, but it would be nice if someone made a tutorial)
-    * [Setting Up Wi-Fi Using the Command Line](../tutorials/setting-up-wifi-using-the-command-line)
-      (requires another connection type first to get to the command line, but it's way
-      easier to enter your passphrase this way)
-* __Bluetooth__
-    * Note: Bluetooth may not work on the first boot. Please reboot if you see "???"
-      after you power on Bluetooth.
-    * [Connecting to the Internet](../tutorials/connecting-to-the-internet-via-bluetooth) tutorial
+Click on the button below to visit our networking page. There are lots of
+choices here; choose the connection option that best suits you and your
+available hardware.
 
 {% include icon.html type="info" %}
 For Raspberry Pi with no display, you must use the wired Ethernet port
 to connect for the first time. You can set up additional connections using the
 `connmanctl` command.
 {: class="alert alert-info"}
 
-Now that you have a network connection, you should
-[connect to your EV3 with SSH](/docs/tutorials/connecting-to-ev3dev-with-ssh)
-if you haven't done so already.
+<a class="btn btn-md btn-primary" href="/docs/networking" target="_blank"><span class="glyphicon glyphicon-globe"></span> Check out the available networking options</a>
```

Comment:
I imagine a user clicking on things in order, reading the content between each button; if they are doing it that way, they wouldn't see the alert until after they had visited the page and presumably configured a connection. Maybe this alert should just be moved to the networking page; what do you think?